Training25/Program.cs

```diff
@@ -3,11 +3,36 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program to check whether the password is strong or not with some criteria.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
+
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      while (true) {
+         Write ("Strong Password\n~~~~~~~~~~~~~~~\nEnter a password: ");
+         var password = ReadLine ();
+         if (string.IsNullOrWhiteSpace (password)) { WriteLine ("Invalid Input!\n"); continue; }
+         bool iWeak = false;
+         if (password.Length < 6) iWeak = PrintError ("Must be atleast 6 characters in length");
+         if (!password.Any (char.IsDigit)) iWeak = PrintError ("Must contain atleast one digit");
+         if (!password.Any (char.IsLower)) iWeak = PrintError ("Must contain atleast one " +
+            "lowercase English character");
+         if (!password.Any (char.IsUpper)) iWeak = PrintError ("Must contain atleast one " +
+            "uppercase English character");
+         if (password.All (char.IsAsciiLetterOrDigit)) iWeak = PrintError ("Must contain " +

```

Comment:
```suggestion
         if (password.All (char.IsLetterOrDigit)) iWeak = PrintError ("Must contain " +
```
This checks unicode characters, but ASCII range is different. Please read about both.